interface/src/avatar/Avatar.cpp

```diff
@@ -543,7 +544,7 @@ void Avatar::postUpdate(float deltaTime) {
     }
 }
 
-void Avatar::render(RenderArgs* renderArgs, const glm::vec3& cameraPosition) {
+void Avatar::render(RenderArgs* renderArgs, render::ScenePointer scene, const Camera& camera) {
```

Comment:
every time we pass a shared pointer like `render::ScenePointer` by value there is a small cost.  There's no reason I can tell not to be passing it by reference as `const render::ScenePointer& scene`